src/components/CommentsList.tsx

```diff
@@ -0,0 +1,45 @@
+import { Comment } from '../types/Comment';
+
+type Props = {
+  comments: Comment[],
+  deleteComments: (commentId: number) => void,
```

Comment:
The function name should be 'deleteComment' as it deletes a single comment, not multiple.